src/base/log.cc

```diff
@@ -177,7 +177,7 @@ void do_log(const LogType type, const Time& gametime, const char* const fmt, ...
 	assert(logger != nullptr);
 
 	// message type and timestamp
-	char buffer_prefix[32];
+	char buffer_prefix[256];
```

Comment:
```suggestion
	char buffer_prefix[sizeof "[24:00:00.000 xxxx] XXXXXXX:"];
```

Reply:
If we want to change the format in the future, even for something small, then we'd need to update two places instead of one. It's just one stack-allocated array with short lifespan so this doesn't have any impact resources-wise; also the gametime can have 3 or more digits in the hour in which case this could overflow. So IMHO better to allow some extra space.